main.js

```diff
@@ -225,3 +227,8 @@ function updateDom(){
 //for all 3 garments find corresponding button and add active class
 //add active image class to images from outfit
 }
+
+function resetBear(){
+  clearInputField(saveOutfit);
+  resetDataModel();
+}
```

Comment:
This function just uses the same functions we already use during save to clear input, clear clothes and background from the bear, reset the buttons, and reset the data model.